backend/location/migrations/0004_auto_20210121_1452.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.13 on 2021-01-21 14:52
+
+import django.contrib.gis.db.models.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('location', '0003_location_name'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='location',
+            old_name='polygon',
+            new_name='multi_polygon',
+        ),
+        migrations.AddField(
+            model_name='location',
+            name='main_polygon',
+            field=django.contrib.gis.db.models.fields.PolygonField(blank=True, geography=True, null=True, srid=4326),
+        ),
+        migrations.AlterField(
+            model_name='location',
+            name='state',
+            field=models.CharField(blank=True, help_text="Points to location's state name", max_length=1024, null=True, verbose_name='Name'),
+        ),
+    ]
```

Comment:
Since you are making changes to a same model and its not deployed to production, you can remove 4 migrations and then create new one so it more organized.

Reply:
Good point. done (locally)!